docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+version: "3.5"
+services:
+
+  db:
+    image: mysql
+    restart: always
+
+    env_file: ./.env
+    ports:
+      - 3308:3306
+
+    environment:
+      - MYSQL_ROOT_PASSWORD=$ROOT_PWD
+      - MYSQL_USER=$MYSQL_USR
+      - MYSQL_PASSWORD=$MYSQL_PWD
+      - MYSQL_DATABASE=$DB_NAME
+
+
+
+
+  app:
+     image: books
+     ports:
+        - 8081:8080
+     environment:
+          SPRING_DATASOURCE_URL: jdbc:mysql://localhost:3308/books?serverTimeZone=UTC
+
```

Comment:
incomplete configuration
ports should be in .env file too
![image](https://github.com/Alex-zzzz/books/assets/49767049/50b39cb8-7bcc-4b45-b4b7-5850459bc823)